### PR TITLE
refactor(ui): sync layout and sidebar theme with figma

### PR DESCRIPTION
**Pull request by grikomsn**
## Description

This PR updates/refactors base layout and sidebar theming to match with Figma spec.

## Changes

- [x] update `tailwindcss` and `daisyui` dependencies (bb0bd3afd9d2fa09ba6456b184531c11789eda63, 8c34a19fffeb0df290104f753ba5999b3ac5431a)
- [x] include tailwind neutral colors (831e2452b106280c24a1a810be56b8b16407e55d)
- [x] add plumbus confetti svg background image (a94af9ce9bac3339924376d6f44b6e11ae5f12dd)
- [x] restructure layout and sidebar, match colors with figma spec (9b2642dea81d6de8efee60697058dbb97304b5f2, 933072b66c953542ace97d2cf256767d0251adb7)

## Screenshots

| Before | After |
|--------|-------|
|image|image|

## Testing Steps

\-

## Links

- https://www.figma.com/file/ajr8SLkAaA08upoIMp6bds/?node-id=0%3A1

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-          <Sidebar />
-          <div className="flex justify-center items-center h-full w-full">
-            {children}
-          </div>
-        </div>
-      </main>
+      {/* eslint-disable-next-line @next/next/no-img-element */}
+      <img
+        src="/confetti.svg"
+        alt="plumbus confetti"
```
Comment (XanderJL) [inline components/Layout.tsx:17]: 😂

**Review by XanderJL — APPROVED** [review]
Not overly familiar with tailwind custom configs, but this was easy enough to follow and looks good :) 🚀

File: components/Layout.tsx [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
+        className="absolute pointer-events-none h-[calc(100vh+180px)] top-0 right-0"
+      />
+
+      <div className="h-full flex z-10">
+        <Sidebar />
+        <main className="flex-grow flex flex-col items-center justify-center">
+          {children}
+          {/*  */}
```
Comment (RyanWarner) [inline components/Layout.tsx:25]: What is this line doing?

**Review by RyanWarner — APPROVED** [review]
😍

File: components/Layout.tsx (hunk 2 above)
Comment (grikomsn) [reply components/Layout.tsx:25]: Mostly for placeholder so when adding/removing lines, git will show the diff replacing the placeholder. But we can remove this if you'd like:

```suggestion
```

**Pull request merged**